pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -1063,3 +1074,13 @@ def test_flip_conj_multispw(sma_mir, tmp_path):
     sma_mir.filename = ms_uv.filename = None
 
     assert sma_mir == ms_uv
+
```

Comment:
Suggest adding this to get the warnings pipeline to pass:
```suggestion
@pytest.mark.filterwarnings("ignore:" + _future_array_shapes_warning)
```